admin/views/tabs/metas/paper-content/media-content.php

```diff
@@ -33,16 +33,15 @@
 	$noindex_option_name = 'noindex-' . $wpseo_post_type->name;
 
 	if ( WPSEO_Options::get( 'is-media-purge-relevant' ) && WPSEO_Options::get( $noindex_option_name ) === false ) {
-		$description = sprintf(
-			/* translators: %1$s expands to the link to the article, %2$s closes the link to the article */
-			esc_html( __( 'By enabling this option, attachment URLs become visible to both your visitors and Google.
-				To add value to your website, they should contain useful information, or they might have a
-				negative impact on your ranking. Please carefully consider this and %1$sread this post%2$s if
-				you want more information about the impact of showing media in search results.', 'wordpress-seo'
-			) ),
-			'<a href="' . esc_url( WPSEO_Shortlinker::get( 'https://yoa.st/2r8' ) ) . '" rel="noopener noreferrer" target="_blank">',
-			'</a>'
-		);
+		$description =
+			__( 'By enabling this option, attachment URLs become visible to both your visitors and Google.', 'wordpress-seo' ) . '<br>' .
+			__( 'To add value to your website, they should contain useful information, or they might have a negative impact on your ranking.', 'wordpress-seo' ) . '<br>' .
```

Comment:
Same as above: When separating this string from the one above, it becomes unclear what 'they' refers to.

Reply:
Commited https://github.com/pedro-mendonca/wordpress-seo/commit/a280dc2908c365375f099835a208d0f9953d27a7 to solve this review.